src/Data/ApiJsonStore.php

```diff
@@ -12,9 +13,19 @@ class ApiJsonStore extends JsonStore
 {
     public function removeClass(Project $project, $name)
     {
-        $path = $this->getFilename($project, $name);
```

Comment:
The signature for `getFilename()` changed - it was easier to just refactor this method to call the parent method and ignore the exception.